src/xcode/ENA/ENAUITests/TraceLocations/ENAUITests_09_TraceLocations.swift

```diff
@@ -35,8 +34,7 @@ class ENAUITests_09_TraceLocations: XCTestCase {
 		app.launch()
 
 		let traceLocationsCardButton = app.buttons[AccessibilityIdentifiers.Home.traceLocationsCardButton]
-		XCTAssertTrue(traceLocationsCardButton.waitForExistence(timeout: .extraLong))
-		traceLocationsCardButton.tap()
```

Comment:
maybe it makes sense to preserve the .extraLong timeout

Reply:
i had the idea to start for all tests with a small timeout, and if it fails, we can increase the timeout afterwards. Until now, Actually it seems that it works with the small timeout (About 30+ runs with no error)